README.md

```diff
@@ -35,6 +35,16 @@ const zipFilePath = await downloadArtifact({
 
 ### Specifying a mirror
 
+To specify another location to download Electron assets from, the following options are
+available:
+
+* `mirrorOptions` Object
+  * `mirror` String (optional) - The base URL of the mirror to download from.
+  * `nightly_mirror` String (optional) - The Electron nightly-specific mirror URL.
+  * `customDir` String (optional) - The name of the directory to download from, often scoped by version number.
+  * `customFilename` String (optional) - The name of the asset to download.
+  * `baseOnly` Boolean (optional) - Whether to download from the base URL only.
```

Comment:
`baseOnly` as an API doesn't make a lot of sense when their aren't any other options called "base".  My suggestion here would be to make it take a `resolveAssetURL` method instead which can then return a full URL if it wants.  This gives folks absolute control over the URL that is returned and removed the strangeness around "what is base" and such